kvs-jwt-verify/verify-jwt.js

```diff
@@ -1,17 +1,15 @@
 import crypto from 'crypto';
 import cf from 'cloudfront';
 
-// updated the original example from below URL to use KVS 
-// https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/example-function-validate-token.html
 
 //Response when JWT is not valid.
 const response401 = {
     statusCode: 401,
     statusDescription: 'Unauthorized'
 };
 
-// Replace the KVS_ID with your KVS ID
-const kvsId = "KVS_ID";
```

Comment:
You remove the KVS_ID from the code, but its still referenced in live 131. You will also need to remove it from line 131.